migration-guides/0.15-Migration-Guide.md

```diff
@@ -1,6 +1,14 @@
 # PureScript 0.15 Migration Guide
 
-## ES modules migration guide 
+## Tooling
+
+- `purescript-psa` does not need to be updated.
+- `spago` needs to be updated to `vT.B.D`.
+- `pulp` needs to be updated to `v16.0.0`.
+- `purs-tidy` needs to be updated to `vT.B.D`.
+- Use the `prepare-0.15` `package-sets` branch for `v0.15.0`-compatible libraries
```

Comment:
At the time of release we'll also release the package set, so I don't think this line should be included